### PR TITLE
Remove `Claim` from Crab

### LINKED ISSUE #870: Remove claim pallet in Crab's next runtime
And the ui will be removed after that, all the unclaimed CRAB tokens will be claimed using new solution, aka. using airdrop tools.

And, the token balances of unclaimed token will need to be migrated to a multisig address for airdrop tool to use.

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Close #870.

~~And there is a referendum on the way.~~
~~To decide how to deal with the un-claims. Burn or move to treasury account.~~
> https://crab.subscan.io/referenda/46

### CODE REVIEW

File: runtime/crab/src/migrations.rs [hunk 1]
```diff
@@ -1,24 +1,48 @@
...
+	RuntimeBlockWeights::get().max_block
 }
 
 pub struct CustomOnRuntimeUpgrade;
 impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 	#[cfg(feature = "try-runtime")]
 	fn pre_upgrade() -> Result<(), &'static str> {
+		let claims_pallet_account = PalletId(*b"da/claim").into_account();
+
+		assert!(Ring::free_balance(&claims_pallet_account) != 0);
```
Comment (AsceticBear) [inline runtime/crab/src/migrations.rs:35]: Should be better to check the `claims_pallet_account` locked value here?

Comment (AurevoirXavier) [reply runtime/crab/src/migrations.rs:35]: `free_balance` means all except reserve.

Comment (AsceticBear) [reply runtime/crab/src/migrations.rs:35]: As my understanding, locked_balance is also part of the free_balance, we just check `!= 0`, what if `locked_balance` isn't `1 * COIN`.

Comment (AurevoirXavier) [reply runtime/crab/src/migrations.rs:35]: I have checked from chain state.

**Review by HackFisher — APPROVED** [review]
Conflicts exists.

**Comment by HackFisher** [comment]
@AurevoirXavier Compile failed

**Pull request merged**